src/AgillicSDK/Agillic.swift

```diff
@@ -363,8 +363,9 @@ public class Agillic : NSObject, SPRequestCallback {
 private class BasicAuth : NSObject, Auth {
     var authInfo: String
     @objc public init(user : String, password: String) {
-        let userPw = user + ":" + password
-        authInfo = "Basic " + userPw.data(using: .utf8)!.base64EncodedString()
+        let authString = "Basic \(user):\(password)"
+        let authData = authString.data(using: .utf8)!
```

Comment:
This one has still has force unwrapping.

Reply:
It is safe as we already made sure user and password has been set before requesting Auth. Additionally: 
_When you use a Unicode encoding (such as .utf8 or .utf16), you can safely force-unwrap. Strings in Swift use Unicode internally, so encoding a string using a Unicode encoding will always succeed._

https://www.objc.io/blog/2018/02/13/string-to-data-and-back/#:~:text=When%20you%20use%20a%20Unicode,Unicode%20encoding%20will%20always%20succeed.